composer/trainer/trainer.py

```diff
@@ -2373,7 +2366,7 @@ def save_checkpoint(self, name: str = 'ep{epoch}-ba{batch}-rank{rank}', *, weigh
             weights_only (bool, optional): See :func:`.save_checkpoint`.
 
         Returns:
-            List[pathlib.Path]: See :func:`.save_checkpoint`.
+            str or None: See :func:`.save_checkpoint`.
```

Comment:
is it me or is this function confusingly named given that there is another `save_checkpoint` and a `_save_checkpoint`. Any way to change the naming here without breaking the interface?

Reply:
Resolving as it looks like duplicate comment (github glitch?)